packages/skin/dist/chip/chip.css

```diff
@@ -21,12 +89,16 @@
 }
 
 button.chip__button {
```

Comment:
This probably needed cascading from the component originally. Should we change it? 

`.chip .chip__button`

Reply:
I believe this won't work because we prefix all button styles with `button.btn` (this is because it can be an anchor or a button)
I will prefix it with `.chip` so that the order the CSS won't cause issues.